packages/components/app/styles/components/side-nav.scss

```
@@ -0,0 +1,205 @@
+:root {
+  /* Sizes: */
+  --token-side-nav-item-padding-horizontal: 12px;
+
+  /* Colors (WIP): */
+  --token-side-nav-foreground-primary: #dedfe3;
+  --token-side-nav-foreground-primary-strong: #fff;
+  --token-side-nav-foreground-faint: #8c909c;
+
+  // Non-active "Links"
+  --token-side-nav-surface-default: #0c0c0e;
+  --token-side-nav-surface-hover: #3b3d45;
+  --token-side-nav-surface-active: #656a76;
+
+  // "Active" "Links"
+  --token-side-nav-selected-surface-default: #656a76;
+  --token-side-nav-selected-surface-hover: #3b3d45;
+  --token-side-nav-selected-surface-active: #656a76;
+}
+
+
+// * SideNav Parent (wrapper) component
+// -------------------------------------------------------------------
+
+.hds-side-nav {
+  display: flex;
+  flex-direction: column;
+  width: 280px;
+  height: 100vh;
+  color: var(--token-side-nav-foreground-primary); // Delete?
+  background: var(--token-side-nav-surface-default);
+}
+
+// Child "wrapper" blocks & child components:
+
+// * Sub-component: Header
+// ---------------------------------------
+.hds-side-nav__header {
+  display: flex;
+  gap: 8px; // TODO: replace with margins
+  align-items: center;
+  padding: 12px;
+  background: var(--token-side-nav-surface-default); // Q: Should this be added here or should it just be transparent?
+
+  .hds-side-nav__header-logo {
+    margin-right: auto; // Q: Make more generic?
+  }
+
+  // TEMP HACK ONLY! (TODO: replace)
+  .hds-disclosure__toggle:not(:hover) * {
+    color: var(--token-side-nav-foreground-primary) !important;
+  }
+}
+
+.hds-side-nav__header-logo {
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+  min-width: 48px;
+  height: 48px;
+  padding: 4px;
+  color: var(--token-side-nav-foreground-primary);
+  background: var(--token-side-nav-surface-default); // Q: Should this be added here or should it just be transparent?
+  border: 1px solid transparent;
+  border-radius: 5px;
+
+  &:focus,
+  &.mock-focus {
+    @include hds-focus-ring-with-pseudo-element();
+  }
+}
+
+// Some styles in cloud-ui, interfere so adding additional selector to increase specificity
+body .hds-side-nav__header-logo {
+  &:hover,
+  &.hover {
+    background: var(--token-side-nav-surface-hover);
+    border-color: transparent;
+  }
+}
+
+body .hds-side-nav__header-logo {
+  &:active,
+  &.active {
+    background: var(--token-side-nav-surface-active);
+    border-color: transparent;
+  }
+}
+
+.hds-side-nav__header-logo-icon {
+  width: auto;
+  height: 35px;
+}
+
+// * Sub-component: Body
+// ---------------------------------------
+.hds-side-nav__body {
+  flex: 1;
+  padding: 12px;
+  overflow: auto;
+}
+
+.hds-side-nav__section {
+  color: var(--token-side-nav-foreground-primary); // Delete?
+  background: var(--token-side-nav-surface-default); // Q: Should this be added here or should it just be transparent?
+
+  + .hds-side-nav__section { margin-top: 28px; }
```

easier to read/interpret (otherwise one could have a doubt that it's `.a + .b` instead of `.a + .a`
```suggestion
  & + & { margin-top: 28px; }
```

---

plus, in Figma seems `24px` to me